README.md

```diff
@@ -1,15 +1,29 @@
 # vets-json-schema [![Build Status](https://travis-ci.org/department-of-veterans-affairs/vets-json-schema.svg?branch=master)](https://travis-ci.org/department-of-veterans-affairs/vets-json-schema)
 
+## Purpose
+Forms on VA.gov use JSON schema to define a common contract for data validation between the front and back ends. This is where those schemas are kept.
+
 ## Development
 
+### Workflow
+1. Clone the repo
+1. Make a branch for your changes
+1. Make the changes needed for your form
+1. Update the version number in `package.json`
+1. Submit a PR
+1. Once that PR is merged
+    1. Create a PR in `vets-website` to update the `vets-json-schema` dependency
+        - This can be done with `yarn update:schema`
+    1. Create a PR in `vets-api` to update the dependency
```

Comment:
Not that this ends up mattering since it's corrected in the readable view, but these are all 1.

Reply:
Yeah, I read somewhere that order lists should all be `1` because it makes modifying them later easier. :man_shrugging: